thread/common/thrprof.c

```diff
@@ -1025,3 +1030,35 @@ omrthread_get_jvm_cpu_usage_info_error_recovery(void)
 		GLOBAL_UNLOCK_SIMPLE(lib);
 	}
 }
+
+intptr_t
+omrthread_get_self_thread_time(omrthread_thread_time_t *threadTime)
```

Comment:
Suggest the function should be named `omrthread_get_thread_times()` for consistency with `omrthread_get_process_times()`.

Reply:
The name `omrthread_get_thread_times` could be misleading that this function can get thread time for an arbitrary  thread?

Reply:
It can't be for any other thread because there is no parameter which could specify that other thread.